### PR TITLE
feat(analytics): support for additional starting objects

### PR DESCRIPTION
**Pull request by winged**
We now also support:
  * Work items and everything "below"
  * Documents and their answers

### CODE REVIEW

File: caluma/caluma_analytics/simple_table.py [hunk 1]
```diff
@@ -740,6 +754,116 @@ def root_fields(self):
         return {**direct_fields, **form_fields, **workitem_fields}
 
 
+class WorkItemsStartingObject(BaseStartingObject):
+    identifier = "workitems"
```
Comment (anehx) [inline caluma/caluma_analytics/simple_table.py:758]:
```suggestion
    identifier = "work_items"
```

**Review by anehx — CHANGES_REQUESTED** [review]
Minor nitpicks, apart from that LGTM as far as I can tell!

**Comment by winged** [comment]
Note: Contains #1737 which should be reviewed first

File: caluma/caluma_analytics/simple_table.py [hunk 2]
```diff
@@ -740,6 +754,116 @@ def root_fields(self):
         return {**direct_fields, **form_fields, **workitem_fields}
 
 
+class WorkItemsStartingObject(BaseStartingObject):
+    identifier = "work_items"
+    label = "Work Items"
+
+    def get_query(self):
+        return self.visibility_source.workitems()
```
Comment (anehx) [inline caluma/caluma_analytics/simple_table.py:762]: Can we rename this to `.work_items()` as well to be consistent?

Comment (winged) [reply caluma/caluma_analytics/simple_table.py]: Done

File: caluma/caluma_analytics/simple_table.py [hunk 3]
```diff
@@ -740,6 +754,116 @@ def root_fields(self):
...
+            ),
+            "meta": MetaField(
+                parent=None,
+                identifier="meta",
+                label="Meta",
+                visibility_source=self.visibility_source,
+            ),
+            "form_id": AttributeField(
+                # TODO can we use "form" as identifier and "form_id" as actual field?
+                # Alternative - join form and present all of the attributes it has, in turn?
```
[the remark this answers is not in the document]
Comment (winged) [reply caluma/caluma_analytics/simple_table.py]: Thought a bit about this - Leaving it with "task_id" / "form_id", so we can add a "Join field" later on with the name "task" or "form" respectively, leading to datasource names like "task.slug" or "task.meta.foo" or similar, without breaking stuff.

**Review by anehx — APPROVED** [review]

**Pull request merged**